wikipron/config.py

```diff
@@ -128,6 +130,41 @@ def _get_process_pron(
             processors.append(functools.partial(re.sub, r"\.", ""))
 
         prosodic_markers = frozenset(["ˈ", "ˌ", "."])
+        mods = frozenset(
+            [
+                "ˡ",
+                "ˍ",
+                "ʲ",
+                "ˠ",
+                "˺",
+                "ː",
+                "˞",
+                "˽",
+                "ˬ",
+                "˖",
+                "ʰ",
+                "ˤ",
+                "˳",
+                "˟",
+                "ⁿ",
+                "ʷ",
+                "˕",
+                "ˌ",
+                "˷",
+                "˔"
+            ]
+        )
+
+        def combining_mods(pron: str) -> str:
+            chars = []
+            for char in pron:
+                # Keep combining characters together
+                if unicodedata.combining(char) or char in mods:
+                    m = chars.pop()
```

Comment:
This `m` is the last character in `chars` that is _not_ a modifier, correct? I was confused by the name `m`. Might be nice to change its name to `last_char` or something, no strong preference though.

Reply:
Good point, I will change that.